base/array.jl

```diff
@@ -1987,3 +1987,139 @@ julia> symdiff([1,2,3],[3,4,5],[4,5,6])
 ```
 """
 symdiff(a, b, rest...) = symdiff(a, symdiff(b, rest...))
+
+## replace/replace! ##
+
+# NOTE: to implement replace! and replace for a new type T, it's enough to define:
```

Comment:
Ok, I will replace `_replace` by `replace` in the recommandation. Until a better place is suggested, I will keep it here though.